src/gpg/contact/trust.py

```diff
@@ -0,0 +1,18 @@
+import sys
+from fn.interact import interact
+
+
+def trust(contact, level):
+    commands = ";trust;;{level};;quit;;".format(level=level).split(";")
```

Comment:
You can build here a list straightaway, instead of doing it with string, format and split.